fast64_internal/oot/oot_collision.py

```diff
@@ -383,11 +383,11 @@ def ootCameraPosToC(camPos):
 		str(camPos.unknown) + ' },\n'
 
 def ootCameraEntryToC(camPos, camData, camPosIndex):
-	return "{ " +\
-		str(camPos.camSType) + ', ' +\
-		('3' if camPos.hasPositionData else '0') + ', ' +\
-		(("&" + camData.camPositionsName()  + '[' + str(camPosIndex) + ']') 
-		if camPos.hasPositionData else "0") + ' },\n'
+	return " ".join((
+		f"{'{'} {camPos.camSType},",
+		f"{'3' if camPos.hasPositionData else '0'},",
+		f"{f'&{camData.camPositionsName()}[{camPosIndex}]' if camPosIndex > 0 else camData.camPositionsName()} {'}'}\n"
+	))
```

Comment:
please change the `f"{'{'}"` stuff to something more sane :sweat_smile: like
```py
	return " ".join((
		"{",
		f"{camPos.camSType},",
		"3" if camPos.hasPositionData else "0",
		",",
		f"&{camData.camPositionsName()}[{camPosIndex}]" if camPosIndex > 0 else camData.camPositionsName(),
		"}\n",
	))
```

also what does the `if camPosIndex > 0` ternary achieve? if it's 0 it just should be `&...[0]`, or can it be < 0

Reply:
``&...[0]`` looks weird imo, using the array name looks cleaner to me but yea it's the same thing so I guess it doesn't matter